.env

```diff
@@ -18,3 +18,4 @@ USER_INFO_COOKIE_NAME=null
 SECURE_ORIGIN_XBLOCK_BOOTSTRAP_HTML_URL=null
 BLOCKSTORE_COLLECTION_UUID=null
 LOGO_TRADEMARK_URL=null
+LIBRARY_LISTING_PAGINATION_PAGE_SIZE=null
```

Comment:
Presumably 2U will have to set this value for our Prod. What is the spec to set it at again? 40?

Reply:
I think we decided that this value is 50